src/Microsoft.IdentityModel.JsonWebTokens/JwtTokenUtilities.cs

```diff
@@ -54,6 +55,45 @@ public class JwtTokenUtilities
         /// </summary>
         public static Regex RegexJwe = new Regex(JwtConstants.JweCompactSerializationRegex, RegexOptions.Compiled | RegexOptions.CultureInvariant, TimeSpan.FromMilliseconds(100));
 
+        internal static Dictionary<string, object> CreateDictionaryFromClaims(IEnumerable<Claim> claims)
+        {
+            if (claims == null)
```

Comment:
since this is an internal method, how about in both case (claims == null) && foreach loop never is entered ( there are no claims ), just return empty dictionary.

Reply:
Made this change.